msgid ""
msgstr "Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-04-08 15:01+0200\n"
"PO-Revision-Date: 2014-04-09 06:29-0400\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: German\n"
"Language: de_DE\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.net\n"

#: pyload/Core.py:182
msgid "Received Quit signal"
msgstr "Signal zum Beenden erhalten"

#: pyload/Core.py:303
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad ist bereits gestartet mit der pid %s"

#: pyload/Core.py:317
#, python-format
msgid "Failed changing group: %s"
msgstr "Fehler beim Ändern der Gruppe: %s"

#: pyload/Core.py:327
#, python-format
msgid "Failed changing user: %s"
msgstr "Fehler beim Ändern des Benutzers: %s"

#: pyload/Core.py:329
msgid "folder for logs"
msgstr "Ordner für Logs"

#: pyload/Core.py:340
msgid "Starting"
msgstr "starte"

#: pyload/Core.py:341
#, python-format
msgid "Using home directory: %s"
msgstr "Benutze Home-Verzeichnis: %s"

#: pyload/Core.py:350
msgid "pycrypto to decode container files"
msgstr "pyCrypto um Containerdateien zu öffnen"

#: pyload/Core.py:353
msgid "folder for temporary files"
msgstr "Ordner für temporäre Dateien"

#: pyload/Core.py:358
msgid "folder for downloads"
msgstr "Downloadordner"

#: pyload/Core.py:361
msgid "OpenSSL for secure connection"
msgstr "OpenSSL für eine sichere Verbindung"

#: pyload/Core.py:365
msgid "Moving old user config to DB"
msgstr "Migriere alte Benutzereinstellungen in die Datenbank..."

#: pyload/Core.py:368
msgid "Please check your logindata with ./pyload.py -u"
msgstr "Bitte überprüfe deine Login-Informationen mit './pyload.py -u'"

#: pyload/Core.py:371
msgid "All links removed"
msgstr "Alle Links entfernt"

#: pyload/Core.py:402
#, python-format
msgid "Downloadtime: %s"
msgstr "Downloadzeit: %s"

#: pyload/Core.py:412
#, python-format
msgid "Free space: %s"
msgstr "Freier Speicher: %sGB"

#: pyload/Core.py:432
msgid "Activating Accounts..."
msgstr "Aktiviere Konten..."

#: pyload/Core.py:438
msgid "Activating Plugins..."
msgstr "Aktiviere Plugins..."

#: pyload/Core.py:441
msgid "pyLoad is up and running"
msgstr "pyLoad ist gestartet und läuft"

#: pyload/Core.py:460
msgid "restarting pyLoad"
msgstr "starte pyLoad neu"

#: pyload/Core.py:464
msgid "pyLoad quits"
msgstr "pyLoad wird beendet"

#: pyload/Core.py:521
#, python-format
msgid "Install %s"
msgstr "Installiere %s"

#: pyload/Core.py:557
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "konnte %(desc)s nicht finden: %(name)s"

#: pyload/Core.py:559
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "konnte %(desc)s nicht erstellen: %(name)s"

#: pyload/Core.py:580
msgid "shutting down..."
msgstr "Beenden..."

#: pyload/Core.py:597
msgid "error while shutting down"
msgstr "Fehler beim Beenden"

#: pyload/Core.py:661
msgid "killed pyLoad from Terminal"
msgstr "pyLoad vom Terminal beendet"

#: pyload/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "Link Datenbank wurde aufgrund inkompatibler Version gelöscht."

#: pyload/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "Link Datenbank konnte nicht konvertiert werden."

#: pyload/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "Datenbank wurde von v2 nach v3 konvertiert."

#: pyload/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "Datenbank wurde von v3 nach v4 konvertiert."

#: pyload/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Konvertiere alte Django DB"

#: pyload/database/FileDatabase.py:45
msgid "finished"
msgstr "Fertig"

#: pyload/database/FileDatabase.py:45
msgid "offline"
msgstr "Offline"

#: pyload/database/FileDatabase.py:45
msgid "online"
msgstr "Online"

#: pyload/database/FileDatabase.py:45
msgid "queued"
msgstr "eingereiht"

#: pyload/database/FileDatabase.py:45
msgid "skipped"
msgstr "übersprungen"

#: pyload/database/FileDatabase.py:45
msgid "waiting"
msgstr "wartend"

#: pyload/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "kurzzeitig offline"

#: pyload/database/FileDatabase.py:45
msgid "starting"
msgstr "starte"

#: pyload/database/FileDatabase.py:45
msgid "failed"
msgstr "fehlgeschlagen"

#: pyload/database/FileDatabase.py:45
msgid "aborted"
msgstr "abgebrochen"

#: pyload/database/FileDatabase.py:45
msgid "decrypting"
msgstr "entschlüsseln"

#: pyload/database/FileDatabase.py:45
msgid "custom"
msgstr "benutzerdefiniert"

#: pyload/database/FileDatabase.py:45
msgid "downloading"
msgstr "downloade"

#: pyload/database/FileDatabase.py:45
msgid "processing"
msgstr "verarbeite"

#: pyload/database/FileDatabase.py:45
msgid "unknown"
msgstr "unbekannt"

#: pyload/database/FileDatabase.py:531 pyload/plugins/addon/IRCInterface.py:74
#: pyload/plugins/addon/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Paket fertiggestellt: %s"

#: pyload/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Nutze SSL Thrift Backend"

#: pyload/manager/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Fehler beim Fernzugriff: %s"

#: pyload/manager/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Starte %(name)s: %(addr)s:%(port)s"

#: pyload/manager/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Konnte Backend %(name)s nicht laden | %(error)s"

#: pyload/webui/app/json.py:57
#, python-format
msgid "waiting %s"
msgstr "Warte %s"

#: pyload/thread/ServerThread.py:35
msgid "SSL certificates not found."
msgstr "SSL Zertifikat nicht gefunden."

#: pyload/thread/ServerThread.py:39
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Entschuldigung! Wir haben den Support für das Starten von %s direkt in pyLoad eingestellt"

#: pyload/thread/ServerThread.py:40
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Du kannst den multithreaded-server benutzen, welcher eine gute Performance, SSL und"

#: pyload/thread/ServerThread.py:41
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "natürlich kannst du weiterhin deine existierende %s mit pyLoads fastcgi server benutzen"

#: pyload/thread/ServerThread.py:42
msgid "sample configs are located in the tools/webserver directory"
msgstr "Beispiel-Konfigurationsdateien befinden sich in dem Verzeichnis 'tools/webserver'"

#: pyload/thread/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "Kann %(server)s nicht benutzen, weil python-flup nicht installiert ist!"

#: pyload/thread/ServerThread.py:56
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Fehler beim Laden vom lightweight Server: %s"

#: pyload/thread/ServerThread.py:57
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Du musst 'bjoern' herunterladen und kompilieren, https://github.com/jonashaag/bjoern"

#: pyload/thread/ServerThread.py:58
msgid "Copy the boern.so to pyload/lib folder or use Setup.py install"
msgstr "Kopiere 'boern.so' in 'pyload/lib' oder benutze 'Setup.py install'"

#: pyload/thread/ServerThread.py:59
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Natürlich musst musst etwas Ahnung von Linux haben und wie man Software kompiliert"

#: pyload/thread/ServerThread.py:63
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Der 'threaded' Server wird wegen Performance Problemen unter Windows benutzt."

#: pyload/thread/ServerThread.py:80 pyload/thread/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "SSL funktioniert mit diesem Server nicht, benutze bitte alternativ den 'threaded' Server"

#: pyload/thread/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Starte den eingebauten Webserver: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Starte den threaded SSL Webserver: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Starte den threaded Webserver: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Starte fastcgi Webserver: %(host)s:%(port)d"

#: pyload/thread/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Starte bjoern Webserver: %(host)s:%(port)d"

#: pyload/webui/app/pyload.py:125
msgid "You dont have permission to access this page."
msgstr "Sie haben nicht die Berechtigung diese Seite zu besuchen."

#: pyload/webui/app/pyload.py:193
msgid "Download directory not found."
msgstr "Downloadverzeichnis nicht gefunden."

#: pyload/webui/app/pyload.py:260 pyload/webui/app/pyload.py:267
msgid "unlimited"
msgstr "unbegrenzt"

#: pyload/webui/app/pyload.py:262 pyload/webui/app/pyload.py:269
msgid "not available"
msgstr "nicht verfügbar"

#: pyload/webui/app/pyload.py:508
msgid "Run pyload.py -s to access the setup."
msgstr "Starte pyload.py -s um auf dieses Setup zuzugreifen."

#: pyload/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Eine Download Verbindung ist fehlggeschlagen, Falle zurück auf eine Verbindung | %s"

#: pyload/thread/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Starte Download: %s"

#: pyload/thread/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Download fertiggestellt: %s"

#: pyload/thread/PluginThread.py:194 pyload/thread/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "Dem Plugin %s fehlt eine Funktion."

#: pyload/thread/PluginThread.py:202 pyload/thread/PluginThread.py:265
#: pyload/thread/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Download abgebrochen: %s"

#: pyload/thread/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Download erneut gestartet: %(name)s | %(msg)s"

#: pyload/thread/PluginThread.py:231 pyload/thread/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "Download ist nicht verfügbar: %s"

#: pyload/thread/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "Download ist vorübergehend nicht verfügbar: %s"

#: pyload/thread/PluginThread.py:237 pyload/thread/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Download fehlgeschlagen: %(name)s | %(msg)s"

#: pyload/thread/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "Verbindungsaufbau zum Host fehlgeschlagen oder die Verbindung wurde zurückgesetzt. Erneuter Versuch in 1 Minute..."

#: pyload/thread/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Download %(name)s wegen %(plugin)s übersprungen"

#: pyload/thread/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Decrypting startet: %s"

#: pyload/thread/PluginThread.py:376 pyload/thread/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Decrypting fehlgeschlagen: %(name)s | %(msg)s"

#: pyload/thread/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Wiederhole %s"

#: pyload/thread/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Holen von Informationen für %(name)s fehlgeschlagen | %(err)s"

#: pyload/manager/AddonManager.py:90 pyload/plugins/Addon.py:102
#, python-format
msgid "Error executing addon: %s"
msgstr "Fehler beim ausführen des addon: %s"

#: pyload/manager/AddonManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Konnte %(name)s nicht aktivieren"

#: pyload/manager/AddonManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Aktivierte Plugins: %s"

#: pyload/manager/AddonManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Deaktivierte Plugins: %s"

#: pyload/manager/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "erneute Verbindung fehlgeschlagen: %s"

#: pyload/manager/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Skript für erneute Verbindung nicht gefunden!"

#: pyload/manager/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "versuche erneute Verbindung"

#: pyload/manager/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Fehler beim Ausführen des Skriptes für die erneute Verbindung!"

#: pyload/manager/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Verbindung erneut aufgebaut, IP: %s"

#: pyload/manager/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Auf dem Gerät ist nicht mehr genügend Speicherplatz vorhanden"

#: pyload/plugins/Account.py:85 pyload/plugins/Account.py:91
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Fehler beim Anmelden mit Konto %(user)s | %(msg)s"

#: pyload/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Falsches Passwort"

#: pyload/plugins/Account.py:240
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "Ihre Zeit %s hat ein falsches Format, nutzen Sie dieses: 1:22-3:44"

#: pyload/plugins/Account.py:266
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "Konto %s hat nicht genügend Traffic, erneuter Versuch in 30 min"

#: pyload/plugins/Account.py:273
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "Konto %s ist abgelaufen, erneuter Versuch in 1Std"

#: pyload/plugins/crypter/SerienjunkiesOrg.py:128
msgid "Downloadlimit reached"
msgstr "Downloadlimit erreicht"

#: pyload/manager/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s hat ein ungültiges Muster."

#: pyload/manager/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Fehler beim Importieren von %(name)s: %(msg)s"

#: pyload/plugins/internal/MultiHoster.py:133
msgid "No Hoster loaded"
msgstr "Kein Hoster geladen"

#: pyload/plugins/account/BitshareCom.py:37
msgid "Activate direct Download in your Bitshare Account"
msgstr "Aktiviere direkten Download in deinem Bitshare Account"

#: pyload/plugins/container/LinkList.py:65
msgid "LinkList could not be cleared."
msgstr "links.txt konnte nicht geleert werden."

#: pyload/manager/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Account-Einstellungen aufgrund neuem Konfigurationsformat gelöscht."

#: pyload/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Authorisierung erforderlich (Benutzername:Passwort)"

#: pyload/plugins/hoster/SimplydebridCom.py:25
#: pyload/plugins/hoster/RealdebridCom.py:41
#: pyload/plugins/hoster/FreeWayMe.py:38 pyload/plugins/hoster/ZeveraCom.py:22
#: pyload/plugins/hoster/UnrestrictLi.py:53
#: pyload/plugins/hoster/Premium4Me.py:28 pyload/plugins/hoster/FastixRu.py:36
#: pyload/plugins/hoster/AlldebridCom.py:38
#: pyload/plugins/hoster/DebridItaliaCom.py:40
#: pyload/plugins/hoster/RPNetBiz.py:26
#: pyload/plugins/hoster/MultiDebridCom.py:41
#: pyload/plugins/hoster/ReloadCc.py:24 pyload/plugins/hoster/RehostTo.py:26
#: pyload/plugins/hoster/PremiumizeMe.py:22
#: pyload/plugins/addon/RPNetBiz.py:43
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Bitte gib deine %s Kontodaten ein oder deaktiviere dieses Plugin"

#: pyload/plugins/hoster/FilesMailRu.py:99
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Es gab HTML-Code in der heruntergeladen Datei (%s)... Umleitungsfehler? Der Download wird neu gestartet."

#: pyload/plugins/hoster/NetloadIn.py:146
#: pyload/plugins/hoster/NetloadIn.py:170
msgid "File temporarily not available"
msgstr "Datei zur Zeit nicht verfügbar"

#: pyload/plugins/hoster/NetloadIn.py:183
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: warte zwischen den Downloads %d s."

#: pyload/plugins/hoster/NetloadIn.py:214
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: Warte auf Captcha %d s."

#: pyload/plugins/hoster/NetloadIn.py:252
msgid "Downloaded File was empty"
msgstr "Die heruntergeladene Datei ist leer"

#: pyload/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "API Schlüssel ungültig"

#: pyload/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Nicht genügend Traffic übrig"

#: pyload/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Traffic überschritten"

#: pyload/plugins/hoster/RapidshareCom.py:100
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Traffic Share (direkter Download)"

#: pyload/plugins/hoster/RapidshareCom.py:127
#: pyload/plugins/hoster/RapidshareCom.py:194
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Von dieser IP Adresse wird bereits heruntergeladen, warte 60 Sekunden"

#: pyload/plugins/hoster/RapidshareCom.py:131
msgid "Invalid Auth Code, download will be restarted"
msgstr "Ungültiger Authorization-Code, Download wird neugestartet"

#: pyload/plugins/hoster/RapidshareCom.py:199
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: Keine freien Slots"

#: pyload/plugins/hoster/RapidshareCom.py:202
msgid "You need a premium account for this file"
msgstr "Du benötigst ein Premium-Konto für diese Datei"

#: pyload/plugins/hoster/RapidshareCom.py:204
msgid "Filename reported invalid"
msgstr "Dateiname als ungültig gemeldet"

#: pyload/plugins/hoster/FileserveCom.py:99
msgid "Parallel download error, now waiting 60s."
msgstr "Parallel-Download Fehler, warte 60s."

#: pyload/plugins/hoster/FileserveCom.py:215
msgid "Not logged in."
msgstr "Nicht eingeloggt."

#: pyload/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Entschlüsselung fehlgeschlagen"

#: pyload/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Für die Datei wurde kein Schlüssel in der URL übertragen"

#: pyload/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Fehler Code:"

#: pyload/plugins/Container.py:68
msgid "File not exists."
msgstr "Datei existiert nicht."

#: pyload/plugins/addon/UpdateManager.py:74
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr "*** Plugins wurden aktualisiert, bitte starte pyLoad neu ***"

#: pyload/plugins/addon/UpdateManager.py:76
msgid "Plugins updated and reloaded"
msgstr "Plugins aktualisiert und neu geladen"

#: pyload/plugins/addon/UpdateManager.py:79
msgid "No plugin updates available"
msgstr "Keine Plugin-Updates verfügbar"

#: pyload/plugins/addon/UpdateManager.py:96
msgid "No Updates for pyLoad"
msgstr "Keine Updates für pyLoad"

#: pyload/plugins/addon/UpdateManager.py:100
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Neue pyLoad Version %s verfügbar  ***"

#: pyload/plugins/addon/UpdateManager.py:101
msgid "***  Get it here: http://pyload.org/download  ***"
msgstr "***  Download hier: http://pyload.org/download  ***"

#: pyload/plugins/addon/UpdateManager.py:104
msgid "Not able to connect server for updates"
msgstr "Konnte keine Verbindung zum Server herstellen, um auf Updates zu prüfen"

#: pyload/plugins/addon/UpdateManager.py:148
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr "Neue Version von %(type)s | %(name)s : %(version).2f"

#: pyload/plugins/addon/UpdateManager.py:157
#: pyload/plugins/addon/UpdateManager.py:162
#, python-format
msgid "Error when updating %s"
msgstr "Fehler beim Aktualisieren von %s"

#: pyload/plugins/addon/UpdateManager.py:162
msgid "Version mismatch"
msgstr "Versionskonflikt"

#: pyload/plugins/addon/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Download fertig: %(name)s @ %(plugin)s "

#: pyload/plugins/addon/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Neue Captcha Anfrage: %s"

#: pyload/plugins/addon/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Beantworte Captcha mit \"c %s Text\""

#: pyload/plugins/addon/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Bitte füge erst dein premium.to Konto hinzu und starte pyLoad neu"

#: pyload/plugins/addon/HotFolder.py:82
#, python-format
msgid "Added %s from HotFolder"
msgstr "%s aus HotFolder hinzugefügt"

#: pyload/plugins/addon/ExtractArchive.py:95
#, python-format
msgid "No %s installed"
msgstr "Kein %s installiert"

#: pyload/plugins/addon/ExtractArchive.py:97
#: pyload/plugins/addon/ExtractArchive.py:102
#, python-format
msgid "Could not activate %s"
msgstr "Konnte %s nicht aktivieren"

#: pyload/plugins/addon/ExtractArchive.py:107
msgid "Activated"
msgstr "Aktiviert"

#: pyload/plugins/addon/ExtractArchive.py:109
msgid "No Extract plugins activated"
msgstr "Keine Entpacken-Plugins aktiviert"

#: pyload/plugins/addon/ExtractArchive.py:121
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Paket %s eingereiht für späteres entpacken"

#: pyload/plugins/addon/ExtractArchive.py:144
#, python-format
msgid "Check package %s"
msgstr "Überprüfe Paket %s"

#: pyload/plugins/addon/ExtractArchive.py:185
#, python-format
msgid "Extract to %s"
msgstr "Entpacke nach %s"

#: pyload/plugins/addon/ExtractArchive.py:200
msgid "No files found to extract"
msgstr "Keine Dateien zum entpacken gefunden"

#: pyload/plugins/addon/ExtractArchive.py:207
msgid "extracting"
msgstr "entpacke"

#: pyload/plugins/addon/ExtractArchive.py:218
msgid "Password protected"
msgstr "Passwortgeschützt"

#: pyload/plugins/addon/ExtractArchive.py:239
msgid "Wrong password"
msgstr "Falsches Passwort"

#: pyload/plugins/addon/ExtractArchive.py:247
#, python-format
msgid "Deleting %s files"
msgstr "Lösche %s Dateien"

#: pyload/plugins/addon/ExtractArchive.py:254
msgid "Extracting finished"
msgstr "Entpacken abgeschlossen"

#: pyload/plugins/addon/ExtractArchive.py:260
msgid "Archive Error"
msgstr "Archiv-Fehler"

#: pyload/plugins/addon/ExtractArchive.py:262
msgid "CRC Mismatch"
msgstr "CRC Fehler"

#: pyload/plugins/addon/ExtractArchive.py:266
msgid "Unknown Error"
msgstr "Unbekannter Fehler"

#: pyload/plugins/addon/ExtractArchive.py:318
msgid "Setting User and Group failed"
msgstr "Setzen von Benutzer und Gruppe fehlgeschlagen"

#: pyload/plugins/addon/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: Port 9666 wird bereits verwendet"

#: pyload/plugins/addon/CaptchaTrader.py:69
#: pyload/plugins/addon/Captcha9kw.py:59
#: pyload/plugins/addon/ExpertDecoders.py:50
#, python-format
msgid "%s credits left"
msgstr "%s Credits verbleibend"

#: pyload/plugins/addon/CaptchaTrader.py:117
msgid "Could not send response."
msgstr "Konnte keine Antwort senden."

#: pyload/plugins/addon/CaptchaTrader.py:135
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "Dein CaptchaTrader Konto hat nicht genügend Credits"

#: pyload/plugins/addon/LinkdecrypterCom.py:43
msgid "Crypter list not found"
msgstr "Liste von Cryptern nicht gefunden"

#: pyload/plugins/addon/LinkdecrypterCom.py:57
msgid "Crypter list is empty"
msgstr "Liste von Cryptern ist leer"

#: pyload/plugins/addon/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Download abgeschlossen: %(name)s @ %(plugin)s"

#: pyload/plugins/addon/Captcha9kw.py:93
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Neue CaptchaID von Upload: %s : %s"

#: pyload/plugins/addon/Captcha9kw.py:129
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "Dein Captcha 9kw.eu Account hat nicht genügend Credits"

#: pyload/plugins/addon/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Installierte Skripte für %s: "

#: pyload/plugins/addon/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script nicht ausführbar:"

#: pyload/plugins/addon/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Fehler in %(script)s: %(error)s"

#: pyload/plugins/addon/ExpertDecoders.py:95
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Dein ExpertDecoders Account hat nicht genügend Credits"

#: pyload/plugins/addon/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Bitte fügen Sie zuerst Ihr rehost.to Konto hinzu und starten Sie pyLoad danach neu"

#: pyload/plugins/addon/PremiumizeMe.py:48
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Bitte füge erst ein gültiges premiumize.me Konto hinzu und starte pyLoad neu."

#: pyload/plugins/addon/CaptchaBrotherhood.py:69
#, python-format
msgid "%d credits left"
msgstr "%d Credits verbleibend"

#: pyload/plugins/Base.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil und Tesseract sind nicht installiert und kein Client ist verbunden für Captcha-Entzifferung"

#: pyload/plugins/Base.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Es konnte in angemessener zeit kein Captcha-Ergebnis durch dies Plugins ermittelt werden."

#: pyload/plugins/Base.py:498 pyload/plugins/Base.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Benutzer und Gruppe setzen fehlgeschlagen: %s"

#: pyload/manager/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "Kein Client verbunden zum eingeben des Captchas"

#: pyload/api/__init__.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Paket %(name)s mit %(count)d Links hinzugefügt"

#: pyload/api/__init__.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "%(count)d Links wurden zum Paket #%(package)d hinzugefügt "

#: pyload/manager/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "Keine JS Engine erkannt. Bitte installiere entweder Spidermonkey, ossp-js, pyv8 oder rhino"

